db/migrate/20200909141146_add_image_display_option_to_editions.rb

```diff
@@ -0,0 +1,15 @@
+class AddImageDisplayOptionToEditions < ActiveRecord::Migration[5.1]
+  EDITIONS_TYPES_WITH_NO_IMAGES = ["CorporateInformationPage", "DocumentCollection", "StatisticalDataSet"].freeze
+
+  def change
+    add_column :editions, :image_display_option, :string, default: "organisation_image"
+
+    editions_with_images = Edition.find_by_sql(["SELECT e.* FROM editions e LEFT OUTER JOIN images ON e.id = images.edition_id WHERE type in ('CaseStudy', 'Consultation', 'DetailedGuide', 'FatalityNotice', 'NewsArticle', 'Publication', 'Speech') AND images.id IS NOT NULL"])
```

Comment:
It would be nice to see this use the same approach as you've used on line 2, so that we can see at a glance which edition types have images and which don't. I.e.

```ruby
EDITIONS_TYPES_WITH_IMAGES = ['CaseStudy', 'Consultation', 'DetailedGuide', 'FatalityNotice', 'NewsArticle', 'Publication', 'Speech'].freeze
# ...
    editions_with_images = Edition.find_by_sql(["SELECT e.* FROM editions e LEFT OUTER JOIN images ON e.id = images.edition_id WHERE type in (#{EDITIONS_TYPES_WITH_IMAGES.to_s}) AND images.id IS NOT NULL"])
```

Reply:
I think this is replaced with another migration later. But I can add it in if it'll be useful for people going forward :)

Reply:
Fair enough - I've made a suggestion in another comment to fix up the migration to this commit so that this file goes away completely! 👍